lldb/source/API/CMakeLists.txt

```diff
@@ -232,7 +231,7 @@ elseif (LLDB_EXPORT_ALL_SYMBOLS)
   )
 
   add_llvm_symbol_exports(liblldb ${exported_symbol_file})
-  set_target_properties(liblldb_exports PROPERTIES FOLDER "lldb misc")
+  set_target_properties(liblldb PROPERTIES FOLDER "LLDB/Misc")
```

Comment:
Should this be `LLDB/API`?

Reply:
This `set_target_properties` is actually redundant: `add_llvm_symbol_exports` already puts it into a folder. It is also `${LLVM_SUBPROJECT_TITLE}/Misc`. I agree `${LLVM_SUBPROJECT_TITLE}/API` would be a better name. I am going to make that change in #89741. Thanks.